tests/config/pytest_modules.yml

```
@@ -1797,6 +1797,10 @@ instrain/profile:
   - modules/nf-core/instrain/profile/**
   - tests/modules/nf-core/instrain/profile/**
 
+interproscan/profile:
+  - modules/nf-core/interproscan/profile/**
+  - tests/modules/nf-core/interproscan/profile/**
```

```suggestion
interproscan:
  - modules/nf-core/interproscan/**
  - tests/modules/nf-core/interproscan/**
```
I'm sorry it had to be the other way around, my bad!